include/gdsb/graph_input.h

```diff
@@ -130,7 +139,7 @@ std::tuple<Vertex, uint64_t> read_graph(std::istream& input,
         if constexpr (GraphParameters::is_weighted())
         {
             string_source = string_position;
-            w = read_ulong(string_source, &string_position);
+            w = read_float(string_source, &string_position);
```

Comment:
does this mean that all graphs are now read as if they have floating point weights? Seems like `read_ulong` is no longer used anywhere then. Maybe remove that code?